model/src/main/proto/arguments.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// The fragment arguments passed to instances of ExitProfileDialogFragment.
+message ExitProfileDialogArguments {
+  // Decides the correct menu item to be highlighted after canceling the ExitProfile dialog.
+  HighlightItem highlight_item = 1;
+}
+
+enum HighlightItem {
+  NONE_UNSPECIFIED = 0;
```

Comment:
This is a bit confusing to me. Is NONE a separate option compared to UNSPECIFIED? They seem to be two different cases that should be handled separately
If thats the case you should have UNSPECIFIED/UNKNOWN as the 0th element, followed by NONE, LAST_CHECKED ....

Reply:
Actually, I only intended to use NONE but protobuf says that the first item of an enum should end with `UNSPECIFIED`.

Reply:
Should we omit this case and simply use the `UNSPECIFIED` case that we don't need to mention in the enum?

Reply:
@vinitamurthi Took inspiration from other enums and refactored it as follows - 
```kotlin
enum HighlightItem {
  // Indicates that none of the items needs to be highlighted.
  ITEM_UNSPECIFIED = 0;

  // Indicates that one of the NavigationDrawerItem needs to be highlighted.
  LAST_CHECKED_MENU_ITEM = 1;

  // Indicates that the Administrator Controls item needs to be highlighted.
  ADMINISTRATOR_CONTROLS_ITEM = 2;
}
```

Please confirm if it looks good?

Reply:
if NONE is a viable option, we should have something like 
```
enum HighlightItem {
  // Indicates that the highlight item was not set
  ITEM_UNSPECIFIED = 0;

  // Indicates that none of the items needs to be highlighted.
  NONE = 1;

  // Indicates that one of the NavigationDrawerItem needs to be highlighted.
  LAST_CHECKED_MENU_ITEM = 2;

  // Indicates that the Administrator Controls item needs to be highlighted.
  ADMINISTRATOR_CONTROLS_ITEM = 3;
}
```

Reply:
Done.